scripts/vaccination_alerts/generate-users-to-email.ts

```diff
@@ -84,7 +108,14 @@ async function main() {
 }
 
 if (require.main === module) {
-  main()
+  const argv = yargs.options({
+    input: {
+      default: DEFAULT_ALERTS_FILE_PATH,
+      description: 'Updated alerts input path.',
+    },
+  }).argv;
+
+  main(argv.input)
```

Comment:
💯  - I'm going to update my script to send test emails after we merge this!

Reply:
missing semicolon (not that I care, but maybe Prettier is not running for you?)

Reply:
I don't think there's a semicolon there because it's a chained operation